packages/test-app/package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "build": "ember build --environment=production",
     "lint:css": "stylelint \"**/*.scss\"",
-    "lint:css:fix": "stylelint \"**/*.scss\" --fix",
+    "lint:css:fix": "concurrently \"pnpm:lint:css -- --fix\"",
```

Comment:
I'd back this change out. We don't use concurrently in our scripts because pnpm provides concurrent tasks and I don't think we have `concurrently` installed here anymore.

Reply:
handled! :heavy_check_mark: